src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  const stylesObject = styles.reduce((acc, style) => {
+    if (style.indexOf(':') === -1) {
+      return acc;
```

Comment:
The check for the presence of a colon is redundant since the filter method on line 12 already ensures that there are no empty style declarations. Instead, you can directly split and trim the property and value without this check.